nerxiv_app/frontend/README.md

```diff
@@ -0,0 +1,26 @@
+# Frontend README
```

Comment:
Deleted this frontend/README.md file, I have other issues describing what to do next.